assets/css/global.css

```diff
@@ -0,0 +1,182 @@
+  src: url("../fonts/rubik-v26-latin-600.woff2") format("woff2"); /* Chrome 36+, Opera 23+, Firefox 39+, Safari 12+, iOS 10+ */
+}
+/* rubik-700 - latin */
+@font-face {
+  font-display: swap; /* Check https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display for other options. */
+  font-family: "Rubik";
+  font-style: normal;
+  font-weight: 700;
+  src: url("../fonts/rubik-v26-latin-700.woff2") format("woff2"); /* Chrome 36+, Opera 23+, Firefox 39+, Safari 12+, iOS 10+ */
+}
+/* rubik-800 - latin */
+@font-face {
+  font-display: swap; /* Check https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display for other options. */
+  font-family: "Rubik";
+  font-style: normal;
+  font-weight: 800;
+  src: url("../fonts/rubik-v26-latin-800.woff2") format("woff2"); /* Chrome 36+, Opera 23+, Firefox 39+, Safari 12+, iOS 10+ */
+}
+
+/* CSS RESET */
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  /* Style & Layout Variables */
+
+  --main-bg-color: rgb(51, 66, 66);
+  --main-accent-color: salmon;
+  --main-text-color: white;
+  --main-width-mobile: 400px;
+}
+
+/* General Styles */
+
+body {
+  font-family: "Rubik", sans-serif;
+  background-color: gray;
+  color: var(--main-text-color);
+  font-weight: 400;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+header {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  background-color: var(--main-bg-color);
+  width: 100%;
+  max-width: var(--main-width-mobile);
+}
+
+main {
+  padding: 2rem;
+  padding-bottom: 7rem;
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  min-height: calc(100vh - 130px);
+  max-width: var(--main-width-mobile);
+  background-color: var(--main-bg-color);
+}
+
+footer {
+  width: 100%;
+  max-width: var(--main-width-mobile);
+  background-color: var(--main-bg-color);
+  position: fixed;
+  bottom: 0;
+}
+
+section {
+  background-color: rgba(22, 85, 57, 0.233);
+  position: relative;
+  padding: 1rem 2rem;
+  border: 2px solid var(--main-text-color);
+  border-radius: 1rem;
+  display: flex;
+  flex-flow: column nowrap;
+  gap: 2rem;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin: 0;
+  padding: 0;
+  font-weight: 200;
+}
+
+ul {
+  list-style: none;
+}
+
+/* Specific Styles */
+
+h1 {
+  font-size: 3rem;
+  text-align: center;
+}
+
+h2 {
+  font-size: 1.6rem;
+  font-weight: 200;
+  margin: 2rem 2rem;
+  margin-left: 0;
+  line-height: 1.4;
+}
+
+h3 {
+  font-size: 1.3rem;
+}
+
+p {
+  font-weight: 200;
+}
+
+button {
+  background-color: transparent;
+  border: 0;
+  cursor: pointer;
+}
+
+hr {
+  margin: 2rem 0;
+  border: 1px solid rgba(255, 255, 255, 0.233);
+}
+
+/* Active Nav Element */
+
+.nav__list .active:hover {
+  color: var(--main-text-color);
+}
+
+.nav__list .active li {
+  background-color: var(--main-accent-color);
+}
+
+/* Active Bookmark Element */
+
+button.active svg {
+  color: var(--main-accent-color);
+}
+
```

Comment:
```css
/* Active Nav Element */

.nav__list .active:hover {
  color: var(--main-text-color);
}

.nav__list .active li {
  background-color: var(--main-accent-color);
}

/* Active Bookmark Element */

button.active svg {
  color: var(--main-accent-color);
}
```

These stylings should be stored in the according component files. :) And if follow BEM syntax you would apply a modifier here: `list__item--active` and `button-bookmark--active`